WORKSPACE

```diff
@@ -40,11 +39,11 @@ http_file(
     urls = ["https://github.com/google/xctestrunner/releases/download/0.2.10/ios_test_runner.par"],
 )
 
-git_repository(
+http_archive(
```

Comment:
I'd recommend not switching to http_archive without updating Bazel to 0.29.x, there were important connection/stream retry fixes in 0.29.x:

- https://github.com/bazelbuild/bazel/pull/9008
- https://github.com/bazelbuild/bazel/pull/9015 (not going to help since we don't have fallback urls here)

Reply:
Thanks! We're on 0.29.1 :)

I did consider listing the same URL multiple times, but read somewhere that it would retry (like 8 times?) regardless. (I didn't read the current source to confirm this.)

Reply:
My bad, didn't notice you've updated Bazel yesterday lol

Listing same URL multiple times might work, but I also haven't checked if it dedups them before attempting the download